### DISCUSSION TITLE
Pro badge not showing

### DISCUSSION THREAD

**Discussion by nurahmed123** [community/community#51737] [Other Feature Feedback, Questions, & Ideas]
I have already claimed the github student pack but the pro badge is not appearing on my profile. What's the process to fixed it

**Comment by meppu** [comment]
If you have pro membership (which is included in github student pack), you can display pro badge from [your github profile](https://github.com/nurahmed123):

click "Edit profile" and enable "Display pro badge"

![image](https://user-images.githubusercontent.com/112294217/229369012-f819e37c-61b6-4435-bb3f-de2186d9cff1.png)
![image](https://user-images.githubusercontent.com/112294217/229369065-4b57e4aa-1b99-4edd-b66d-7615d76a2ec2.png)
